src/main/scala/xiangshan/backend/fu/CSR.scala

```diff
@@ -502,6 +502,7 @@ class CSR(implicit p: Parameters) extends FunctionUnit with HasCSRConst with PMP
     GenMask(XLEN - 2, 36) | // WPRI
     GenMask(35, 32)       | // SXL and UXL cannot be changed
     GenMask(31, 23)       | // WPRI
+    GenMask(16, 15)       | // XS is ready-only
```

Comment:
"read-only", not "ready-only". Also in the commit message.

Reply:
Thank you very much.